src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/EthAddressInput/index.tsx

```diff
@@ -16,18 +18,32 @@ import { styles } from 'src/routes/safe/components/Balances/SendModal/screens/Co
 
 const useStyles = makeStyles(styles)
 
-export interface EthAddressProps {
+export interface AddressBookInputProps {
```

Comment:
nitpick: the component is called `EthAddressInput` but the interface for props is `AddressBookInputProps`, could you please make the name consistent?